corehq/apps/tour/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+import json_field
+
+
+class GuidedTours(models.Model):
+    web_user = models.CharField(max_length=80, null=False)
+    tours = json_field.JSONField(
```

Comment:
is this the tours they need to see? or the tours they have seen? (using a specific name might be better depending on how this is going to be used)